content/en/blog/_posts/2024-05-13-gateway-api-v1.1/index.md

```diff
@@ -0,0 +1,258 @@
+GRPCRoute. We're also introducing some new experimental features, including
+session persistence and client certificate verification.
+
+## What's new
+
+### Graduation to Standard
+This release includes the graduation to Standard of four eagerly awaited features.
+This means they are no longer experimental concepts; inclusion in the Standard
+release channel denotes a high level of confidence in the API surface and
+provides guarantees of backward compatibility. Of course, as with any other
+Kubernetes API, Standard Channel features can continue to evolve with
+backward-compatible additions over time, and we certainly expect further
+refinements and improvements to these new features in the future.
+For more information on how all of this works, refer to the
+[Gateway API Versioning Policy](https://gateway-api.sigs.k8s.io/concepts/versioning/).
+
+
+#### [Service Mesh Support](https://gateway-api.sigs.k8s.io/mesh/)
+Service mesh support in Gateway API allows service mesh users to use the same
+API to manage ingress traffic and mesh traffic, reusing the same policy and
+routing interfaces. In Gateway API v1.1, routes (such as HTTPRoute) can now have
+a Service as a `parentRef`, to control how traffic to specific services behave.
+For more information, read the 
+[Gateway API service mesh documentation](https://gateway-api.sigs.k8s.io/mesh/)
+or see the
+[list of Gateway API implementations
+](https://gateway-api.sigs.k8s.io/implementations/#service-mesh-implementation-status).
+
+As an example, one could do a canary deployment of a workload deep in an
+application's call graph with an HTTPRoute as follows:
+
+```
+apiVersion: gateway.networking.k8s.io/v1
+kind: HTTPRoute
+metadata:
+  name: color-canary
+  namespace: faces
+spec:
+  parentRefs:
+    - name: color
+      kind: Service
+      group: ""
+      port: 80
+  rules:
+  - backendRefs:
+    - name: color
+      port: 80
+      weight: 50
+    - name: color2
+      port: 80
+      weight: 50
+```
+      
+This would split traffic sent to the `color` Service in the `faces` namespace
+50/50 between the original `color` Service and the `color2` Service, using a
+portable configuration that's easy to move from one mesh to another.
+
+
+#### [GRPCRoute](https://gateway-api.sigs.k8s.io/guides/grpc-routing/)
+If you are already using the experimental version of GRPCRoute, we recommend holding
+off on upgrading to the standard channel version of GRPCRoute until the
+controllers you're using have been updated to support GRPCRoute v1. Until then,
+it is safe to upgrade to the experimental channel version of GRPCRoute in v1.1
+that includes both v1alpha2 and v1 API versions.
+
+
+#### [ParentReference Port](https://gateway-api.sigs.k8s.io/reference/spec/#gateway.networking.k8s.io%2fv1.ParentReference)
+The `port` field was added to ParentReference, allowing you to attach resources
+to Gateway Listeners, Services (experimental), or other parent resources
```

Comment:
Is binding to Service still experimental now that Service Mesh Support is promoted to standard?
```suggestion
to Gateway Listeners, Services, or other parent resources
```